book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] It is required to have setter and getter methods for the `:rentals`. It could be simply added to the already written methods for title and author.
```
  attr_accessor :title, :author, :rentals
```